buildpacks/nodejs-npm-install/tests/snapshots/test_skip_build_scripts_from_buildplan.snap

```diff
@@ -36,4 +36,7 @@ source: test_support/src/lib.rs
   - Skipping as pruning was disabled by a participating buildpack
 - Configuring default processes
   - Skipping default web process (no start script defined)
+
+! Experimental feature used - buildpack plan configuration (this feature may change unexpectedly in the future)
```

Comment:
Suggestion (not blocking): I think we should tell people specifics of where the warning is coming from. Either list the key and file here or earlier. 

```
! Warning: Experimental configuration `com.heroku.buildpacks.nodejs.actions` found in `project.toml`.
! This feature may change unexpectedly in the future
```

Reply:
Fixed in 415f79c2